packages/website-frontend/test/unit/services/session-storage.service.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { SessionStorageService } from "../../../src/services/session-storage.service";
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+
+describe(SessionStorageService.name, () => {
+  let sut: SessionStorageService;
+  let sessionStorageMock: Storage;
+
+  beforeEach(() => {
+    sut = new SessionStorageService();
```

Comment:
Same here. What is the value in adding a session storage mock, when you can also just set the sessionStorage you want in the real browser?